CHANGELOG.md

```diff
@@ -9,19 +9,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Legend - label trimming when special characters are present in the labels
 
+### Changed
+- `<ChartWrapper \>` - moved all necessary chart wrapper css in ChartWrapper component
+
 ## [0.3.2] - 2020-08-18
 ### Fixed
-- ReadMe - update scatter & line props usage
-- Utils - update sort function props to be more flexible
+- ReadMe - updated scatter & line props usage
+- Utils - updated sort function props to be more flexible
```

Comment:
no need to introduce unnecessary changes like these, when both present and past tenses are okay.

also, writing git messages in present tense is considered a [normal convention](https://stackoverflow.com/a/3580764/158111) (as `git` itself promotes). and combined with the the past tense headers like `Fixed`, `Changed` in the CHANGELOG itself, it's perfectly normal to write it as:

```
Did:
- utils - update ...
```

for "did update" makes more sense than, "did updated".

Reply:
LOL I looked over react-labs CHANGELOG and wanted to be consistent as we used both. You need me to revert?